source/_integrations/vodafone_station.markdown

```diff
@@ -0,0 +1,52 @@
+---
+title: Vodafone Station
+description: Instructions on how to integrate Vodafone Station routers into Home Assistant.
+ha_category:
+  - Presence Detection
+ha_release: '2023.08'
+ha_domain: vodafone_station
+ha_config_flow: true
+ha_codeowners:
+  - '@paoloantinori'
+  - '@chemelli74'
+ha_iot_class: Local Polling
+ha_platforms:
+  - device_tracker
+ha_ssdp: false
+ha_integration_type: integration
+---
+
+The Vodafone Station integration allows you to control your [Vodafone Station](https://www.vodafone.it/privati/area-supporto/assistenza-dispositivi/vodafone-station.html) based router.
```

Comment:
is there an English page to be linked available?

Reply:
Unfortunately not.